packages/edit-donation/src/donation-form-edit-donation.ts

```diff
@@ -241,6 +260,11 @@ export class DonationFormEditDonation extends LitElement {
         customAmountSpan = 3;
         break;
       case 4:
+        if (minimalView) {
+          columnCount = 4;
+          customAmountSpan = 0;
+          break;
+        }
         columnCount = 3;
         customAmountSpan = 2;
```

Comment:
BLOCKER? Looks like any values for `columnCount` and `customAmountSpan` that are set in the `if` block will be immediately wiped out by the assignments here.

Reply:
hi thanks for checking, that's the behavior we are going for as the original config shows the multi-line amount options + custom input. 👍